packages/magic/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "@tsconfig/svelte/tsconfig.json",
```

Comment:
Wanted checking with @aaronbarnardsound on this comment

Reply:
I don't think we want to extend from the base since it is quite different and there may be conflicts. The `core` and `common` packages both use this svelte specific ts config.